.github/workflows/publish-docs.yml

```diff
@@ -90,17 +91,19 @@ jobs:
         uses: peaceiris/actions-hugo@v2
         with:
           hugo-version: '0.79.1'
-      - name: Build Hugo docs
+      - name: Build searcher
```

Comment:
Both steps build-search as a dependency, so this step should not be needed, right?